docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md

```diff
@@ -106,7 +106,23 @@ spec:
   ...
 ```
 
+#### With EKS Support
+
+The pre-requisities for IPv6.... are enabled by default. However, if you want to use some of the optional features of EKS (see [here](eks/enabling.md) for more information on what these are) then you will need to enable these features via the configuration file. For example:
```

Comment:
```suggestion
The pre-requisities for IPv6 are not enabled by default. If you want to create IPv6 enabled clusters using CAPA then you will need to enable these permissions via the configuration file. For example:
```

Reply:
And depending on a previous comment this may need to be changed.

Reply:
@Skarlso Update date this once discussions are concluded.